NEWS.md

```diff
@@ -12,7 +12,10 @@ New language features
     `findall`, `findfirst`, `argmin`/`argmax` and `findmin`/`findmax` to work with these
     objects, returning the index of matching non-missing elements in the parent ([#31008]).
 
-  * `inv(::Missing)` has now been added and returns `missing` ([#31408]).
+  * `inv(::Missing)` has now been added and returns `missing` ([#31451]).
+
+  * `nextfloat(::BigFloat, n::Int)` and `prevfloat(::BigFloat, n::Int)` have now been added
```

Comment:
nitpick, if you happen to re-push before merge: `n::Integer`

Reply:
```suggestion
  * `nextfloat(::BigFloat, n::Integer)` and `prevfloat(::BigFloat, n:: Integer)` methods have been added
```

Reply:
@narendrakpatel is in class and will push this modification  when he gets home.

Reply:
or -- just go ahead and make the change -- we all agree

Reply:
I am back. I will make the changes right away.

Reply:
do you know how to "squash"? I don't.

Reply:
For now, I will just use `git commit --amend` but you can squash commit in the following way:
* `git rebase -i HEAD~n`
* replace `pick` with `squash`

Reply:
You don't have to do anything. It can be squashed by the person doing the merge.

Reply:
(But yes, it can be done by you as well with the appropriate git incantations.)